pkg/helm/client/client.go

```diff
@@ -79,6 +82,7 @@ func (c namespaceClientConfig) ConfigAccess() clientcmd.ConfigAccess {
 	return nil
 }
 
+// NewRESTClientGetter ...
```

Comment:
While these functions are exported and in a non-internal package, we don't really consider them public, so I don't think we _need_ GoDoc comments.

It's obviously okay to add them, but if we're adding them, let's make sure they have a full description of the function.